src/modules/Channel/components/TypingIndicator.tsx

```diff
@@ -15,27 +14,31 @@ export interface TypingIndicatorTextProps {
   members: Member[];
 }
 
-export const TypingIndicatorText: React.FC<TypingIndicatorTextProps> = ({ members }: TypingIndicatorTextProps) => {
-  const { stringSet } = useContext(LocalizationContext);
-  if (!members || members.length === 0) {
-    return '';
-  }
+export const TypingIndicatorText = ({ members }: TypingIndicatorTextProps) => {
```

Comment:
can we keep return type?

Reply:
I've changed this as a component (text -> component).

Are there any rules for type specification in components? It seems fragmented. It would be good to decide on this now.

const Component: React.FC<Props> = (props) => { ... };
const Component = (props: Props) => { ... } // Inference as JSX.Element;
const Component = (props: Props): JSX.Element => { ... }
const Component = (props: Props): ReactElement => { ... }
Personally, I prefer option 2.
By the way, JSX.Element is simply an interface that inherits from ReactElement (they are the same type)

Reply:
We don't have any. I prefer 2 too :)